OmsAgent/Fairfax/omsagent_ff.py

```diff
@@ -731,7 +731,7 @@ def log_and_exit(hutil, operation, exit_code = 1, message = ''):
     if hutil is not None:
         hutil.do_exit(exit_code, operation, exit_status, str(exit_code), message)
     else:
-        update_status_file(operation, exit_code, exit_status, message)
+        update_status_file(operation, str(exit_code), exit_status, message)
```

Comment:
I agree with the principle of that; however, I think we should keep the specific message for error code 1.

This error can be generated for several scenarios, which is why it's not provided in our [error code table](https://github.com/Azure/azure-linux-extensions/tree/master/OmsAgent#error-codes-and-their-meanings), but the most common is that curl is not installed. Until the INSTALL_CURL error code change is added to the extension, we should inform the customer here to check for dependencies.

I will make the change to remove the logic for the other error codes, which have not reoccured in the Kusto error logs recently.